deploy-board/deploy_board/settings.py

```diff
@@ -230,6 +230,7 @@
 
 TELETRAAN_DISABLE_CREATE_ENV_PAGE = True if os.getenv("TELETRAAN_DISABLE_CREATE_ENV_PAGE", "false") == "true" else False
 TELETRAAN_REDIRECT_CREATE_ENV_PAGE_URL = os.getenv("TELETRAAN_REDIRECT_CREATE_ENV_PAGE_URL", None)
+TELETRAAN_CLUSTER_READONLY_FIELDS = os.getenv("TELETRAAN_CLUSTER_READONLY_FIELDS", "spiffe_id").split(",")
```

Comment:
need to support default `None` in case the customer does not use spiffe id

Reply:
Making this default to `None` would be a change in behavior for customers.
Previously `spiffe_id` was hard-coded in a bunch of places to be a readonly field, this reduces how many times it's typed into the code, but maintains the same behavior as before.

If we change behavior then upgraders who use a `spiffe_id` field would need to update their configs to keep it read-only, or else risk having their own users be allowed to edit that field unexpectedly.

I think the change in behavior could be it's own diff, and come with the needed documentation/upgrade notes and version bump. This diff enables that change.

Reply:
Thanks, I agree about minimizing changing the behavior in this diff.